Led.cpp

```diff
@@ -1,33 +1,33 @@
 #include "Led.h"
 
-Led::Led() : Pin(Pin) {
+Led::Led() : PIN(0) {
 }
 
-Led::Led(unsigned pin) : Pin(pin) {
+Led::Led(unsigned pin) : PIN(pin) {
     //	pinMode(pin, OUTPUT);
 }
 
 void Led::setBrightness(unsigned percent) {
-    //  percent = map(percent, 0, 100, 0, 255);
+    percent *= 2.55;
     //	analogWrite(led, percent);
     brightness = percent;
 }
 
 void Led::turnOn() {
     //  analogWrite(led, brightness);
-    flag = 1;
+    flag = true;
 }
 
 void Led::turnOff() {
     //	digitalWrite(pin, LOW);
-    flag = 0;
+    flag = false;
 }
 
 void Led::blink(unsigned n, unsigned delay1, unsigned delay2) {
 
-    if (flag == 1) {
+    if (flag == true) {
 
-        for (unsigned i = 0; i <= n; i++) {
+        for (unsigned i = 1; i <= n; i++) {
```

Comment:
since `i<=n` is never used just omit it.
`for( unsigned i = i;; i++)`
It will be clear to people reading the code the the break condition must be within the loop then.